test/fv/utils.js

```diff
@@ -37,25 +37,6 @@ const getTLSArgs = () => {
 
     return '';
 };
-const getPeerAddresses = () => {
```

Comment:
Did you intend to remove these two functions in the same commit? Just asking because we usually keep commits to one thread of work, or at least mention in the commit notes if some other stuff was done. It's fine in this case since it is minor, just double-checking.

Reply:
Yes, my bad, forgot to include this in the description. These two functions are no longer used.